### DISCUSSION TITLE
music/euclidean/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#3] [General]
# music/euclidean/

https://shanenull.com/music/euclidean/

**Comment by shane0** [comment]
| Time Period   | Country/Region | Traditional Rhythm Name | Rhythm Pattern                   | Step Count | Fill Count | Rotation |
| ------------- | -------------- | -----------------------|---------------------------------|------------|------------|----------|
| Medieval      | Europe         | Brevis/Longa            | long-short-short                 | 3          | 2          | 0        |
| Medieval      | Middle East    | Iqa'                    | various                          | various    | various    | various  |
| Renaissance   | Europe         | Pavan                   | long-short-short-short-long-long | 5          | 3          | 0        |
| Baroque       | Europe         | Sarabande               | short-long-short-long            | 4          | 2          | 0        |
| Classical     | Europe         | Minuet                  | short-long-short-long-short-long | 6          | 3          | 0        |
| Romantic      | Europe         | Waltz                   | strong-weak-weak                 | 3          | 1          | 0        |
| Modern        | Latin America | Samba                   | strong-weak-weak-weak            | 4          | 2          | 0        |
| Modern        | Africa         | Kpanlogo                | strong-weak-strong-weak-weak     | 5          | 3          | 0        |
| Modern        | India          | Teental                 | strong-weak-medium-weak          | 4          | 2          | 0        |
| Modern        | Japan          | Shima Uta               | various                          | various    | various    | various  |

**Comment by shane0** [comment]
| Time Period   | Country/Region | Traditional Rhythm Name |
| ------------- | -------------- | -----------------------|
| Medieval      | Europe         | Brevis/Longa            |
| Medieval      | Middle East    | Iqa'                  |
| Renaissance   | Europe         | Pavan                   |
| Baroque       | Europe         | Sarabande               |
| Classical     | Europe         | Minuet                  |
| Romantic      | Europe         | Waltz                   |
| Modern        | Latin America | Samba                   |
| Modern        | Africa         | Kpanlogo                |
| Modern        | India          | Teental                 |
| Modern        | Japan          | Shima Uta               |
